docs/limitations.md

```diff
@@ -242,12 +242,16 @@ See more documentation at
 
 #### `docker --device`
 
-host device support (`docker run --device`) is not supported. This could
-be mapped to the quite similar QEMU `-device` option to enact device
-pass-through if that particular device is supported by VFIO, but the QEMU
-option requires precise knowledge of which sort of device is being
-mapped, and that information is not necessarily easily available from
-the docker `--device` command.
+Support has been added to pass devices using [Virtual Function I/O(VFIO)](https://www.kernel.org/doc/Documentation/vfio.txt) 
+passthrough. Devices that support the Input–Output Memory Management Unit(IOMMU)
```

Comment:
Another missing space before the opening bracket.

Reply:
fixed